jax/experimental/jax2tf/jax2tf.py

```diff
@@ -683,6 +784,21 @@ def _conv_general_dilated(lhs, rhs, window_strides, padding, lhs_dilation,
       lhs, rhs, window_strides, padding, lhs_dilation, rhs_dilation,
       dimension_numbers, feature_group_count, batch_group_count, lhs_shape,
       rhs_shape, precision)
+
+  tf_fun, tf_padding, tf_data_format, tf_dilation = _can_use_tf_conv(
+      lhs, rhs, window_strides, padding, lhs_dilation, rhs_dilation,
+      dimension_numbers, feature_group_count, batch_group_count, out_shape
+  )
+  if tf_fun is not None:
+    result = None
+    if tf_fun is tf.nn.convolution:
```

Comment:
This feel brittle. Wouldn't it be better to just have _can_use_tf_conv return the result, and then you don't have to split the logic between that function and this one?